check_yum

```diff
@@ -332,17 +332,20 @@ class YumTester:
 		if number_updates == 0:
 			status = OK
 			message = "0 Updates Available"
+			perfdata = "updates=0"
```

Comment:
I think this (and following) should rather read something like "non-security-updates=" or so - otherwise it's ambiguous whether or not this number includes security updates.